packages/core/src/Table/stories/TableSection/TableSettings.tsx

```diff
@@ -421,7 +422,7 @@ const SettingsDialog = ({
 
 export const TableSettings = () => {
   const tableColumns = useMemo(() => getColumns(), []);
-  const [popperOpen, setPopperOpen] = useState(false);
```

Comment:
Shouldn't we open the popper only for applitools tests?

Reply:
Don't we need it open to run the a11y checks also? (Not sure if we can also make it open for those checks)

Reply:
Right, I forgot about that. Maybe the `play` property can be used for that: https://storybook.js.org/docs/writing-stories/play-function.

Reply:
Agreed. We should about having popper/panels opened by default in our samples.
The samples are docs-first - it's strange for the user to see them opened by default, and these "poppers" typically (this one doesn't seem too bad) hyjack the page focus/scroll